frontend/src/context/layers/composite_data.ts

```diff
@@ -84,7 +84,7 @@ export const fetchCompositeLayerData: LazyLoader<CompositeLayerProps> =
         key,
         aggregation,
         importance,
-        invert: Boolean(invert),
```

Comment:
This "should" be doing the same thing. Can you explain the bug you were seeing?

Reply:
The error is actually on the JSON side.

"invert": "True" should be "invert": true

And

"invert": "False" should be "invert": false

If you prefer your option and it works well, that's ok, we would simply need to update the type:

    invert?: boolean | undefined; ->    invert?: 'False' | 'True' | undefined;

Reply:
ahh thanks @ericboucher. fixing the error in the config is definitely better. will close this PR